conf/test_full.config

```diff
@@ -0,0 +1,33 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/variantmtb -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Full test profile'
+    config_profile_description = 'TEST: Query all supported knowledgebases and input types using a reference genome identifier and specific cancer type. Some KBs might need access credentials.'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input  = "${projectDir}/tests/csv/input_SNPs-CNAs.csv"
+
+    // Genome references
+    genome = 'GRCh37'
```

Comment:
Why do you use GRCh37? Is it possible to update to 38?

Reply:
this is still the same vcf as the `test` profile, which used GRCh37. Also civic bulk queries only work with GRCh37 AFAIK. It also works with GRCh38, but I though it is better to keep the test data slim.

Reply:
Maybe we can note this somewhere so its not forgotten :) Its not important now but I think at some point we should update the test data!